assets/css/admin.scss

```diff
@@ -78,8 +78,8 @@ a.wpjm-activate-licence-link:active {
 
 			&::before {
 
-				@extend .jm-icon;
-				content: "\e803";
+				@include jm-icon()
+					content: "\e803";
```

Comment:
There are some unexpected tabs.

Reply:
Ahh, that was `lint:css` after I forgot the `;` above. Fixed in 07732a9188b3e0684ac6d1be47db268804a76624